source/conf.py

```diff
@@ -15,6 +15,8 @@
 import sys
 import os
 
+import sys; sys.setrecursionlimit(1500)
```

Comment:
It would be good to have a comment here (or in commit msg) that explains why we need such a high recursion limit.

Reply:
Thanks @mliszcz! It was needed while rearranging ToCs (to fix 'not included in any toc' warning). However, the final arrangement does not require it anymore. I will commnet-out the line and do merge.

Reply:
```suggestion
# import sys; sys.setrecursionlimit(1500)
```